scripts/ci/lib.sh

```diff
@@ -694,7 +694,15 @@ image_prefetcher_system_await() {
 _image_prefetcher_prebuilt_await() {
     case "$CI_JOB_NAME" in
     *qa-e2e-tests)
+        if [[ "${JOB_NAME:-unknown}" =~ interop ]]; then
```

Comment:
Just for my curiosity: where is `JOB_NAME` being set? I wonder if we could get this knowledge from $CI_JOB_NAME instead? So that this could be a simpler `case` statement rather than nested `case/if`?